### ISSUE TITLE
restlet version is too old.

### ISSUE THREAD

**Issue by Shun-Prog** [netsuite-upload-org/netsuite-upload#51]
https://github.com/netsuite-upload-org/netsuite-upload/blob/27074b651a44443a397f53906676a4a5bfbfcb0d/netSuiteRestlet/vscodeExtensionRestlet.js#L14

When i tried push to FileCabinet from VScode,  restlet version error is shown.

I thought the initial version of restlet should be updated 1.0.2 to 1.2.4

**Comment by Shun-Prog** [comment]
#52

**Comment by dennysutanto** [comment]
@Shun-Prog, I don't think it will work, there is a checking here which required it to be 1.0.2
https://github.com/netsuite-upload-org/netsuite-upload/blob/da474286811fec6c841a9c66a0987ec02565c52c/helpers/netSuiteRestClient.js#L86
